packages/wallet-standard/src/wallet.ts

```diff
@@ -179,7 +178,7 @@ export class GlowWallet implements Wallet {
   };
 
   #disconnect: DisconnectMethod = async () => {
-    await window.glow.disconnect();
+    await this.#glow.disconnect();
```

Comment:
We can remove this, pending the decisions on https://github.com/glow-xyz/glow-js/issues/95